packages/wallet/src/rollup-aggregator.ts

```diff
@@ -210,111 +237,185 @@ export class RollupAggregator extends SimpleServer {
    */
   private async requestFaucetFunds(
     signedTransaction: SignedTransaction
-  ): Promise<SignedTransactionReceipt> {
-    if (!isFaucetTransaction(signedTransaction.transaction)) {
-      throw Error('Cannot handle non-Faucet Request in faucet endpoint')
-    }
-    const messageSigner: Address = this.signatureVerifier.verifyMessage(
-      serializeObject(signedTransaction.transaction),
-      signedTransaction.signature
-    )
-    if (messageSigner !== signedTransaction.transaction.requester) {
-      throw Error('Faucet requests must be signed by the request address')
-    }
+  ): Promise<SignedStateReceipt> {
+    try {
+      if (!isFaucetTransaction(signedTransaction.transaction)) {
+        throw Error('Cannot handle non-Faucet Request in faucet endpoint')
+      }
+      const messageSigner: Address = this.signatureVerifier.verifyMessage(
+        serializeObject(signedTransaction.transaction),
+        signedTransaction.signature
+      )
+      if (messageSigner !== signedTransaction.transaction.sender) {
```

Comment:
So I think that this check should instead be something like:
```
if (messageSigner !== HARDCODED_FAUCET_AUTHENTICATED_ADDRESS) { throw error }
```

Because all we need to know is if Uniswap's twitter bot authenticated the recipient of the faucet funds. Other than that we just send blindly to whatever the address is that Uniswap told us to send to

Reply:
Yep, will update when we have that :+1: just want to avoid spending time guessing what that’ll look like to then change it anyway.